apps/system/js/statusbar.js

```diff
@@ -19,43 +19,46 @@
 'use strict';
 
 (function(exports) {
+  const AVG_ICON_SIZE = 22;
```

Comment:
I think we should move forward with this super simple approach, land the patch.
And then later we can optimize the `calculateMinimized` logic :)